### PR TITLE
introduce f1 score

### PR DESCRIPTION
**Pull request by Efefefef**
introduce f1score
remove commented printouts
set train_model_on_coco notebook to actively choose learning rate and freeze layers
model_filename = f'{experiment}.h5'
clear cell outputs

### CODE REVIEW

File: data/out/sample_experiment/layer1/results.json
Comment (imtezcan) [inline data/out/sample_experiment/layer1/results.json:1]: this results.json file already includes results for all layers, why is it in `/layer1/` folder? shouldn't it be under `/sample_experiment/` ?

File: src/algonauts/models/model_loaders.py [hunk 1]
```diff
@@ -9,5 +10,5 @@ def load_vgg16():
 
 
 def load_from_file(model_filename, transform_image):
-    model = tf.keras.models.load_model(model_filename)
+    model = tf.keras.models.load_model(model_filename, custin_objects={'F1Score': tfa.metrics.F1Score})
```
Comment (imtezcan) [inline src/algonauts/models/model_loaders.py:13]: should be "custom_objects" I guess ? :)

Comment (Efefefef) [reply src/algonauts/models/model_loaders.py]: this is weird, i tested and it didn't complain like without passing it, fixed now

File: data/out/sample_experiment/layer1/results.json
Comment (Efefefef) [reply data/out/sample_experiment/layer1/results.json:1]: true

**Review by imtezcan — APPROVED** [review]
